python/test/t_CubaIntegration_std.py

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env python
+
+import openturns as ot
+import math
+import openturns.testing as ott
+
+ot.TESTPREAMBLE()
+
+# Testing four algorithms on a function from R^3 to R.
+f = ot.SymbolicFunction(["x", "y", "z"], ["sin(x) * cos(y) * exp(z)"])
+lbIntegration = ot.Point((0.0, 0.0, 0.0))
+ubIntegration = ot.Point((1.0, 1.0, 1.0))
+routines = ["cuhre", "divonne", "suave", "vegas"]
+valueRef = -math.sin(1.0) * (math.cos(1.0) - 1.0) * (math.e - 1.0)
+
+for r in routines:
+    algoC = ot.CubaIntegration(r)
+    value = algoC.integrate(f, ot.Interval(lbIntegration, ubIntegration))
+    ott.assert_almost_equal(value[0], valueRef, 1.0e-3, 1.0e-3)
+
+# Testing three algorithms on a function from R^3 to R^2.
+f = ot.SymbolicFunction(
+    ["x", "y", "z"],
+    [
+        "exp(-(x * x + y * y + z *z))",
+        "(x / 2 * x / 2 + y / 3 * y / 3 + z / 4 * z / 4 - 1.125)",
+    ],
+)
+lbIntegration = ot.Point((0.0, 0.0, 1.0))
+ubIntegration = ot.Point((2.0, 3.0, 4.0))
+routines = ["cuhre", "divonne", "vegas"]
```

Comment:
does the suave method work here ?

Reply:
It seems to be quite inaccurate for these integrands, as many many more integrand evaluations would be needed to reach a good accuracy. I added a comment about this in the test file.